public/javascripts/financial-data.js

```diff
@@ -0,0 +1,54 @@
+document.getElementById("myBtn").addEventListener("click", function(){
+    const start = document.getElementById('startInput').value
+    const end = document.getElementById('endInput').value  
+    const apiUrl = `http://api.coindesk.com/v1/bpi/historical/close.json?start=${start}&end=${end}`
+    printChart(apiUrl);
+});
+
+
+
+document.getElementById("currency").addEventListener("change", function () {
+    console.log(document.getElementById("currency").value)
+    const changeCurrency = document.getElementById("currency").value
+    const apiUrl = `http://api.coindesk.com/v1/bpi/historical/close.json?function=${changeCurrency}`
```

Comment:
Hey Matthew! in your link here, you're not sending the value of the changeCurrency to the currency from your select.